packages/css/src/blockquote/README.md

```diff
@@ -0,0 +1 @@
+# Blockquote
```

Comment:
Please add some initial docs in this PR already.

- Citaat ([bron](https://www.amsterdam.nl/schrijfwijzer/moeilijke-woorden/))
- Bedoeld om een pagina te verfraaien (?) ([bron](https://www.amsterdam.nl/digitalservices/designrichtlijnen/typografie-stylesheet/))
- Heeft een breedte van een halve of hele tekstkolom (same)
- Voegt automatisch dubbele aanhalingstekens toe.
- Misschien is in [The Components Gallery](https://component.gallery/components/quote/) nog wat te vinden.